docs: Update list of build dependencies (#75)
GreptimeTeam/docs: docs/developer-guide/get-started.md

### Build Dependencies

- Git (optional)
  - Clone the source from

> - C/C++ Toolchain: provides basic tools for compiling and linking. This is available as build-essential on ubuntu and similar name on other platforms.

Reviewer: ```suggestion
- C/C++ Toolchain: provides basic tools for compiling and linking. This is available as `build-essential` on ubuntu and similar name on other platforms.
```

> - C/C++ Toolchain: provides basic tools for compiling and linking. This is available as build-essential on ubuntu and similar name on other platforms.

Reviewer 2: ```suggestion
- C/C++ Toolchain: provides essential tools for compiling and linking. This is available either as `build-essential` on ubuntu or a similar name on other platforms.
```
